File: app/src/main/java/com/amaze/filemanager/fragments/preference_fragments/AdvancedSearchPref.java
package com.amaze.filemanager.fragments.preference_fragments;

import android.os.Bundle;
import android.preference.PreferenceFragment;

import com.amaze.filemanager.R;

/**
 * @author Emmanuel
 *         on 8/5/2017, at 11:30.
 */

public class AdvancedSearchPref extends PreferenceFragment {

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        // Load the preferences from an XML resource
        addPreferencesFromResource(R.xml.advancedsearch_prefs);
    }

}

File: app/src/main/java/com/amaze/filemanager/ui/fragments/preference_fragments/AdvancedSearchPref.kt
/*
 * Copyright (C) 2014-2021 Arpit Khurana <arpitkh96@gmail.com>, Vishal Nehra <vishalmeham2@gmail.com>,
 * Emmanuel Messulam<emmanuelbendavid@gmail.com>, Raymond Lai <airwave209gt at gmail.com> and Contributors.
 *
 * This file is part of Amaze File Manager.
 *
 * Amaze File Manager is free software: you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation, either version 3 of the License, or
 * (at your option) any later version.
 *
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 *
 * You should have received a copy of the GNU General Public License
 * along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */

package com.amaze.filemanager.ui.fragments.preference_fragments

import android.os.Bundle
import androidx.preference.PreferenceFragmentCompat
import com.amaze.filemanager.R

/** @author Emmanuel on 8/5/2017, at 11:30.
 */
class AdvancedSearchPref : PreferenceFragmentCompat() {

    override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
        // Load the preferences from an XML resource
        addPreferencesFromResource(R.xml.advancedsearch_prefs)
    }
}
